### ISSUE TITLE
Search by filename using Regular expression not apparently working

### ISSUE THREAD

**Issue by Jeremy7701** [linuxmint/nemo#3090]
```
 * Nemo version 5.4.3
 * Is issue with desktop or windowed nemo?
 * Distribution - (LMDE5)
 * 64 bit
 ```

**Issue**
Searching **filenames** using Regular expressions are not apparently working?
Or are do Regular expressions only apply when searching by **content**?

**Steps to reproduce**

Searching a folder (which contains many EPUB titles) with `^The` for filenames starting with "The"  doesn't produce any output if the **Re** option is selected.

**Expected behaviour**

**Other information**

**Comment by mtwebster** [comment]
It's only for content (the button is in the content row only also).

Wildcards work (`The*`).

**Comment by actionless** [comment]
could it be reopened as a feature-request?

that would be really nice to have such possibility

**Comment by Tuckertcs** [comment]
> could it be reopened as a feature-request?
> 
> that would be really nice to have such possibility

Bumping, as I also would like this feature. It's surprising that regex searching _is_ implemented, just not for file names.

**Comment by ZebulonGlitch** [comment]
I just found this issue after I saw regex and wow, nemo for the win!  But then it wouldn't do anything....

Now I to am a bit dumbfounded that regex is there but doesn't work on file names...  :-/  I frankly wouldn't expect Nemo to scan file content, there are much better ways to index content and do searches properly.  But finding file names containing spaces?  That's pretty meat and potatoes territory...
